client/landing/subscriptions/components/site-list/styles.scss

```diff
@@ -74,6 +74,19 @@ $max-list-width: 1300px;
 					}
 				}
 			}
+
+			.p2-label {
+				display: inline-block;
+				text-align: center;
+				width: 36px;
+				height: 20px;
+				background: #2a6aee;
+				color: $studio-green-0;
+				font-size: 12;
+				border-radius: 4px;
+				margin-left: 8px;
+				line-height: 20px;
+			}
```

Comment:
Since the same label would be used for Comments list as well, I wonder if it would make sense to move the styling to more general SCSS file that is shared across both lists. We could consider `client/landing/subscriptions/styles.scss` for instance.